Utilities/aliceHLTwrapper/src/WrapperDevice.cxx

```diff
@@ -174,7 +174,7 @@ void WrapperDevice::Run()
         mMaxTimeBetweenSample=sampleTimeDiff;
     }
     mLastSampleTime=duration.count();
-    if (duration.count()-mLastCalcTime>fLogIntervalInMs) {
+    if (duration.count()-mLastCalcTime>1000) {
```

Comment:
The device doesn't have log interval at all now. It is stored for each channel individually. Channel has `GetRateLogging()` that returns the interval. Since the code above only looks at the device and not at any specific channel, more significant refactoring would be necessary there to make sense.